js/app.js

```diff
@@ -0,0 +1,34 @@
+$(function() {
+
+  $("#submit-btn").click(getPicture);
+
+
+  function getPicture() {
+   var city = $("#city-type").val();
```

Comment:
@aaronbatsss consider transform the case of the input value. So as that, it matches the case of the condition you are matching against.